assets/js/components/setup-module.js

```diff
@@ -97,7 +97,7 @@ class SetupModule extends Component {
 
 		let blockedByParentModule = false;
 
-		const { modules } = global.googlesitekit;
+		const { modules } = global._googlesitekitLegacyData;
```

Comment:
This looks like a replacement that was missed from a while back.
I believe it should be this:
```suggestion
		const modules = getModulesData();
```
(may need to import that utility)

Reply:
Done